src/components/Calculator.js

```diff
@@ -1,45 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-    this.handleClick = this.handleClick.bind(this);
-  }
+function Calculator() {
+  const [state, setState] = useState({});
+  const { next, total } = state;
+  const handleClick = (event) => {
+    setState((state) => calculate(state, event.target.textContent));
+  };
```

Comment:
### Optional
- [ ] I highly suggest that you avoid using the `function` keyword instead you can use the arrow function as a new feature of ES6 to make your code more modern and readable you can simply do:
```
const Calculator = () => (
      <div>
      </div>
);
```

Reply:
@DeliceLydia  Thanks for your suggestion I tried it several times but it gives an error which is "Function component is not a function declaration". This is an ESLint error. 
Please let me know how can I resolve this error.

Reply:
To fix this error try running this command
` npx eslint . --fix`